data/config-schema/src/main/java/net/corda/schema/configuration/DatabaseConfig.java

```diff
@@ -11,4 +11,8 @@ private DatabaseConfig() {
     public static final String JDBC_URL = "database.jdbc.url";
     public static final String DB_POOL_MAX_SIZE = "database.pool.max_size";
     public static final String DB_POOL_MIN_SIZE = "database.pool.min_size";
+    public static final String DB_POOL_IDLE_TIMEOUT = "database.pool.idleTimeout";
+    public static final String DB_POOL_MAX_LIFETIME = "database.pool.maxLifetime";
+    public static final String DB_POOL_KEEP_ALIVE_TIME = "database.pool.keepaliveTime";
```

Comment:
as you've not capitalised the 'a' in keepalive, this should be`DB_POOL_KEEPALIVE_TIME`

Reply:
Good catch, addressed.